tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -3227,7 +3227,7 @@ TEST_F(EnergyPlusFixture, FanCoil_FanSystemModelCyclingFanMode)
     Sim4PipeFanCoil(state, FanCoilNum, ZoneNum, ZoneNum, FirstHVACIteration, QUnitOut, QLatOut);
     // expect fan speed 3 and near full air and water flow and meet capacity
     EXPECT_EQ(thisFanCoil.SpeedFanSel, 3);
-    EXPECT_NEAR(thisFanCoil.PLR, 0.970, 0.001);
+    EXPECT_NEAR(thisFanCoil.PLR, 0.961, 0.001);
```

Comment:
Fixing the fan calcs resulted in slightly different PLRs for the fancoil. It's still selecting the same speed number and the load is still passing the test, so I think this is a legitimate change here and below several places.

Reply:
Yeah I don't have any problem with these changes.